{
  "uid" : "a3345bf98b7b7b96",
  "name" : "returns index of first element in next filled range",
  "fullName" : "Contract: Arrays findUpperBound Array with gap.returns index of first element in next filled range",
  "historyId" : "7826e5a66487325b014dfeef6ad64dc6",
  "time" : {
    "start" : 1635555029825,
    "stop" : 1635555030803,
    "duration" : 978
  },
  "description" : "",
  "descriptionHtml" : "",
  "status" : "broken",
  "statusMessage" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a067b3d0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
  "statusTrace" : "Error: Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a067b3d0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))\n                    at Context.<anonymous> (test/utils/Arrays.test.js:69:43)\n                    at processImmediate (node:internal/timers:464:21)",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "suite",
    "value" : "Contract: Arrays findUpperBound Array with gap"
  }, {
    "name" : "testClass",
    "value" : "Contract: Arrays findUpperBound Array with gap"
  }, {
    "name" : "testMethod",
    "value" : "returns index of first element in next filled range"
  }, {
    "name" : "package",
    "value" : "Contract: Arrays findUpperBound Array with gap"
  }, {
    "name" : "resultFormat",
    "value" : "allure1"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ {
      "uid" : "8fc561d848a34cee",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a2856e50>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554962423,
        "stop" : 1635554963425,
        "duration" : 1002
      }
    }, {
      "uid" : "44eb1a47d7f29c8d",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a04288b0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554932774,
        "stop" : 1635554933366,
        "duration" : 592
      }
    }, {
      "uid" : "6c712d847c00894f",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f238be90340>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554921433,
        "stop" : 1635554922032,
        "duration" : 599
      }
    }, {
      "uid" : "8c58e4b48297cef8",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f238be8ce50>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554916890,
        "stop" : 1635554917484,
        "duration" : 594
      }
    }, {
      "uid" : "527c1d7e536b999f",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a1840730>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554889187,
        "stop" : 1635554890173,
        "duration" : 986
      }
    }, {
      "uid" : "dd1fee2b0169a6dc",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a06772b0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554852689,
        "stop" : 1635554853284,
        "duration" : 595
      }
    }, {
      "uid" : "a82e42417f632d27",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0681280>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554840469,
        "stop" : 1635554841062,
        "duration" : 593
      }
    }, {
      "uid" : "e350257377627981",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a042d700>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635554836874,
        "stop" : 1635554837469,
        "duration" : 595
      }
    } ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 2,
        "skipped" : 0,
        "passed" : 0,
        "unknown" : 0,
        "total" : 2
      },
      "items" : [ {
        "uid" : "4ca86e60a4784d73",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/6//#testresult/4ca86e60a4784d73",
        "status" : "broken",
        "statusDetails" : "HH110: Invalid JSON-RPC response received: <html>\n                <head><title>502 Bad Gateway</title></head>\n                <body>\n                <center><h1>502 Bad Gateway</h1></center>\n                <hr><center>nginx/1.18.0 (Ubuntu)</center>\n                </body>\n                </html>\n                ",
        "time" : {
          "start" : 1635551256918,
          "stop" : 1635551256975,
          "duration" : 57
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "a3345bf98b7b7b96.json",
  "parameterValues" : [ ]
}